src/vmm/src/devices/virtio/virtio_block/event_handler.rs

```diff
@@ -60,20 +63,11 @@ impl MutEventSubscriber for VirtioBlock {
         }
 
         if self.is_activated() {
-            let queue_evt = self.queue_evts[0].as_raw_fd();
-            let rate_limiter_evt = self.rate_limiter.as_raw_fd();
-            let activate_fd = self.activate_evt.as_raw_fd();
-            let maybe_completion_fd = match self.disk.file_engine {
-                FileEngine::Async(ref engine) => Some(engine.completion_evt().as_raw_fd()),
-                FileEngine::Sync(_) => None,
-            };
```

Comment:
For reviewers: Can you double check it is ok to replace this with just a u32 ?

Reply:
I think it's fine because it will be unwrapped and return if it is not an async engine IO :)
https://github.com/firecracker-microvm/firecracker/blob/main/src/vmm/src/devices/virtio/virtio_block/device.rs#L482